source/tests/system/nirfsa_driver_api_tests.cpp

```diff
@@ -21,38 +40,340 @@ class NiRFSADriverApiTests : public Test {
+  auto configure_clock = client::configure_ref_clock(stub(), session, RefClockSourceRangeTable::REF_CLOCK_SOURCE_RANGE_TABLE_ONBOARD_CLOCK_STR, 10e6);
+  auto configure_reference_level = client::configure_reference_level(stub(), session, "", 0);
+  auto configure_acquisition_type = client::configure_acquisition_type(stub(), session, AcquisitionTypeRangeTable::ACQUISITION_TYPE_RANGE_TABLE_IQ);
+  auto configure_number_of_samples = client::configure_number_of_samples(stub(), session, "", true, 1000);
+  auto configure_iq_rate = client::configure_iq_rate(stub(), session, "", 1e6);
 
-  auto close_response = client::close(stub(), init_response.vi());
+  EXPECT_SUCCESS(session, configure_clock);
+  EXPECT_SUCCESS(session, configure_reference_level);
+  EXPECT_SUCCESS(session, configure_acquisition_type);
+  EXPECT_SUCCESS(session, configure_number_of_samples);
+  EXPECT_SUCCESS(session, configure_iq_rate);
+}
+
+TEST_F(NiRFSADriverApiTests, ConfigureGettingStartedSpectrum_Succeeds)
+{
+  auto session = init_session(stub(), PXI_5663E);
+  auto configure_clock = client::configure_ref_clock(stub(), session, std::string("OnboardClock"), 10e6);
+  auto configure_reference_level = client::configure_reference_level(stub(), session, "", 0);
+  auto configure_acquisition_type = client::configure_acquisition_type(stub(), session, AcquisitionTypeRangeTable::ACQUISITION_TYPE_RANGE_TABLE_SPECTRUM);
+  auto configure_frequency_start_stop = client::configure_spectrum_frequency_start_stop(stub(), session, "", 990e6, 1010e6);
+  auto configure_resolution_bandwidth = client::configure_resolution_bandwidth(stub(), session, "", 10e3);
+  auto number_of_spectral_lines = client::get_number_of_spectral_lines(stub(), session, "");
 
-  EXPECT_SUCCESS(init_response);
-  EXPECT_SUCCESS(close_response);
+  EXPECT_SUCCESS(session, configure_clock);
+  EXPECT_SUCCESS(session, configure_reference_level);
+  EXPECT_SUCCESS(session, configure_acquisition_type);
+  EXPECT_SUCCESS(session, configure_frequency_start_stop);
+  EXPECT_SUCCESS(session, configure_resolution_bandwidth);
+  EXPECT_SUCCESS(session, number_of_spectral_lines);
+  EXPECT_EQ(4974, number_of_spectral_lines.number_of_spectral_lines());
 }
+
+TEST_F(NiRFSADriverApiTests, GetDeviceResponse_ReturnsResponseData)
+{
+  auto session = init_session(stub(), PXI_5663E);
+  auto response = client::get_device_response(stub(), session, "", DeviceResponse::DEVICE_RESPONSE_DOWNCONVERTER_IF_RESPONSE);
+
+  const auto EXPECTED_SIZE = 81;
+  EXPECT_SUCCESS(session, response);
+  EXPECT_THAT(response.frequencies(), Each(Not(Eq(0))));
+  EXPECT_THAT(response.phase_response(), Each(Eq(0)));
+  EXPECT_THAT(response.magnitude_response(), Each(Eq(0)));
+  EXPECT_EQ(EXPECTED_SIZE, response.frequencies().size());
+  EXPECT_EQ(EXPECTED_SIZE, response.phase_response().size());
+  EXPECT_EQ(EXPECTED_SIZE, response.magnitude_response().size());
+}
+
+// TODO: AB#1639825. This currently fails because of an off-by-one error.
```

Comment:
Does that this mean that this test should be skipped or commented out for now then?

Reply:
Oh nevermind, these are system tests so they're run manually.

Reply:
It looks like the gtest standard is a DISABLED_ prefix. I can do that.